.browserslistrc

```diff
@@ -1 +1 @@
-> 1% or IE 11
+> 0.5%
```

Comment:
This change is necessary since ESBuild doesn't support all transpilations for IE11. 

Admittedly, I thought we'd long dropped support for IE11 in this project, since we already had done so in our other projects.

Technically this is a breaking change, so we'd probably want to note it in the CHANGELOG and bump the major version if we move forward with it.